index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="box header">
+            <span>My Logo</span>
+            <span><img src="img/Enabled.png"></span>
+        </div>
```

Comment:
- [ ] I think it would be better to use semantic HTML5 tags, so I suggest using `<header class="box header">` tag instead of  `<div class="box header">` 👌

- [ ] Kindly make sure this logo is actually a link. You may surround the logo with an `<a>` tag. 👌

> This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] I suggest wrap The hamburger/menu button with a `nav` tag 💯
> The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

For example, your code should be something like this 💡
```
<header>
  <a href="#">My Logo</a>
     <nav>
         <img src="img/Enabled.png" />
     </nav>
</header>
```